### PR TITLE
S2t2 populate component

### PR DESCRIPTION
**Pull request by nico-dev-it**

### CODE REVIEW

File: src/ReadingPanel/EmailBody/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const EmailBody = (props) => {
+    const emailData = props.emailDataById
+
+
+    return (
+        <div>
+            <p className=" font-weight-bold h4">{emailData.subject}</p>
+            <p>{emailData.body}</p>
+
```
Comment (Georgeoak03) [inline src/ReadingPanel/EmailBody/index.js:9]: No need for space 'ere

File: src/ReadingPanel/CloseButton/index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const CloseButton = (props) => {
+    const clickResetView = () => {
+        props.setReadingPanelDisplay(false);
+    }
+        return (
+            <div className="d-flex flex-row justify-content-end">
+                <button onClick={clickResetView} id="closeButton" type="button" className="btn btn-secondary mt-2 d-md-none">X Close</button>
+            </div>
+
```
Comment (Georgeoak03) [inline src/ReadingPanel/CloseButton/index.js:9]: No need for space 'ere

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+
+import './App.css';
+import EmailCardList from "./EmailCardList/EmailCardList.js";
+import Header from './Header/Header.js'
+import SideBar from './SideBar/SideBar.js'
+import DisplayReadingPanel from "./ReadingPanel";
+import {useState, useEffect} from "react";
+
+
```
Comment (timyop) [inline src/App.js:9]: no need for 2 spaces

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    useEffect(() => {
+        fetchEmailById()
+    }, [])
+
+    useEffect(()=> {
+        fetchEmailById()
+        },[emailSearchId])
+
+
```
Comment (timyop) [inline src/App.js:37]: two spaces

File: src/ReadingPanel/EmailBody/index.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+const EmailBody = (props) => {
+    const emailData = props.emailDataById
+
```
Comment (Georgeoak03) [inline src/ReadingPanel/EmailBody/index.js:3]: No need for space 'ere

File: src/App.js [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+        const jsonEmailData = await emailData.json()
+        setAllEmailSnippets(jsonEmailData.data)
+    }
+
+    useEffect(() => {
+        fetchAllEmailData()
+    }, [])
+
+
+
```
Comment (timyop) [inline src/App.js:49]: two spaces

File: src/ReadingPanel/index.js [hunk 7]
```diff
@@ -5,19 +5,12 @@ import './style.css'
...
         <div className={(props.readingPanelDisplay  ? ' ' : 'd-none') + " col-sm-12  vh-100% col-md-7"}>
-            <EmailHeader emailDataById={props.emailDataById}/>
-            <EmailBody emailDataById={props.emailDataById}/>
+            <EmailHeader emailDataById={props.emailDataById} />
+            <EmailBody emailDataById={props.emailDataById} />
         </div>
     )
 }
 
 export default DisplayReadingPanel
```
Comment (timyop) [inline src/ReadingPanel/index.js:18]: extra space not needed

File: src/ReadingPanel/EmailHeader/index.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+    const headerDateHuman = headerDate.toLocaleDateString("en-GB")
+
+    return (
+        <div className="emailHeader d-flex row justify-content-between font-weight-bold pt-2">
+            <div className="header col-8 d-flex flex-column">
+            <p className="headerName h5 font-weight-bold">{headerData.name}</p>
+            <p className="headerEmailAddress">{headerData.email}</p>
+            </div>
+            <div className="headerDate col-4 d-flex justify-content-end h6 font-weight-bold">
+            <p>{headerDateHuman}</p>
```
Comment (timyop) [inline src/ReadingPanel/EmailHeader/index.js:14]: these p might need to be indented

File: src/ReadingPanel/index.js [hunk 9]
```diff
@@ -0,0 +1,18 @@
+import EmailHeader from "./EmailHeader";
```
Comment (timyop) [inline src/ReadingPanel/index.js:1]: take out these semi-colons

**Review by Georgeoak03 — CHANGES_REQUESTED** [review]
Just some spacing

**Review by Georgeoak03 — APPROVED** [review]
Lovely Stuff

**Review by timyop — APPROVED** [review]

**Pull request merged**